webapp/channels/src/components/admin_console/server_logs/logs.tsx

```diff
@@ -137,23 +153,35 @@ export default class Logs extends React.PureComponent<Props, State> {
     };
 
     render() {
-        const content = this.props.isPlainLogs ? (
+        const ToggleView = (
+            <div className='logs-banner'>
+                <Toggle
+                    onText='Formatted'
+                    offText='Plain'
+                    toggled={this.state.isLocalFormattedView}
+                    onToggle={this.toggleMode}
+                />
+            </div>);
+        const content = this.state.isLocalFormattedView ? (
             <>
-                <div className='banner'>
-                    <div className='banner__content'>
-                        <FormattedMessage {...messages.bannerDesc}/>
+                <div className='logs-banner'>
+                    <div className='banner'>
+                        <div className='banner__content'>
+                            <FormattedMessage {...messages.bannerDesc}/>
+                        </div>
                     </div>
+                    <button
+                        type='submit'
+                        className='btn btn-primary'
+                        onClick={this.reloadPlain}
+                    >
+                        <FormattedMessage
+                            id='admin.logs.ReloadLogs'
+                            defaultMessage='Reload Logs'
+                        />
+                    </button>
                 </div>
-                <button
-                    type='submit'
-                    className='btn btn-primary'
-                    onClick={this.reloadPlain}
-                >
-                    <FormattedMessage
-                        id='admin.logs.ReloadLogs'
-                        defaultMessage='Reload Logs'
-                    />
-                </button>
+                {ToggleView}
```

Comment:
Can we place the toggle next to the reload logs button please?

Reply:
Thanks for your comments @hanzei . I have updated the code base. Can you please review it again?